src/ucs/sys/module.c

```diff
@@ -241,8 +256,13 @@ static void ucs_module_load_one(const char *framework, const char *module_name,
 
     ucs_module_trace("loading module '%s' with mode 0x%x", module_name, mode);
 
+    status = ucs_string_alloc_path_buffer(&module_path, "module_path");
```

Comment:
can we use alloc_formatted_path here as well?

Reply:
It'll re-allocate the path buffer each iteration, and it's the only place with a `ucs_string_alloc_path_buffer()` followed by a for loop

Reply:
ok